pandas/core/internals/blocks.py

```diff
@@ -2338,6 +2354,22 @@ def to_dense(self):
         # expects that behavior.
         return np.asarray(self.values, dtype=_NS_DTYPE)
 
+    def to_native_types(
+        self, slicer=None, na_rep=None, date_format=None, quoting=None, **kwargs
+    ):
+        """
+        We need to pick DatetimeBlock's version, but the inheritance structure
+        would use ExtensionBlock's verison
+        """
+        return DatetimeBlock.to_native_types(
+            self,
+            slicer=slicer,
+            na_rep=na_rep,
+            date_format=date_format,
+            quoting=quoting,
+            **kwargs
+        )
+
```

Comment:
Can this go up around L2282 as just `to_native_types = DatetimeBlock.to_native_types`?

Reply:
Yes, that's cleaner, thanks